iexfinance/tests/test_altdata.py

```diff
@@ -23,7 +23,7 @@ def test_crypto_quote_list(self):
             get_crypto_quote(["BTCUSDT", "BAD"])
 
     def test_crypto_quote_json(self):
-        data = get_crypto_quote("BTCUSDT")
+        data = get_crypto_quote("BTCUSDT", output_format='json')
```

Comment:
This can be removed for now...set your output format to ``json`` and it should pass without the parameter. Longer term solution is better as described in review comment